### PR TITLE
28 update environment setup for geofabrics

### PR DESCRIPTION
**Pull request by rosepearson**
Updated environment file for Casey to review:

## Developer Checklist
- [x] Make code change
- [x] Update documentation
  - [x] Readme

## Reviewer Checklist
- [x] Check new code for code smells
- [x] Check if documentation needs updating
  - [x] Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Digital-Twins
 
+## Setup
+Setup a conda environment to run in using the following command run from the repository folder:
```
Comment (caseyli92) [inline README.md:4]: great, thanks for documenting how to set up the environment.

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - geopandas>=0.10.2 # Needs to be explicit otherwise the dependencies required to use sjoin are not installed
+  - rasterio # Needs to be explicit as wheel fails to build if insalled as a dependency of geofabrics
+  - python-pdal # Not on PyPI so needs to be added here and not through pip geofabrics
+  - geoapis>=0.2.8
+  - psycopg2 # Try remove in future
+  - bs4 # Try remove in future
+  - validators  # Try remove in future
+  - sqlalchemy
+  - geoalchemy2
+  - pip:
```
Comment (caseyli92) [inline environment.yml:19]: I think you missed pytest, otherwise good to go.

**Review by caseyli92 — APPROVED** [review]
looks good, can you please add pytest.

File: environment.yml (hunk 2 above)
Comment (rosepearson) [reply environment.yml:20]: good point! Added now

File: README.md (hunk 1 above)
Comment (rosepearson) [reply README.md:4]: Thanks :)

**Pull request merged**